rskj-core/src/main/java/co/rsk/net/BlockCache.java

```diff
@@ -46,4 +46,8 @@ public void addBlock(Block block) {
     public Block getBlockByHash(byte[] hash) {
         return blockMap.get(new Keccak256(hash));
     }
+
+    public void clear() {

```

Comment:
This is not being used i think

Reply:
This method was called but not it's caller, so I'l remove all the chain of methods used in the `BlockStore::clear` interface